recipe/build.sh

```diff
@@ -4,18 +4,28 @@ set -exv
 make generate --jobs ${CPU_COUNT}
 
 # Duplicate lists because of https://bitbucket.org/icl/magma/pull-requests/32
-export CUDA_ARCH_LIST="sm_35,sm_50,sm_60,sm_70,sm_80"
-export CUDAARCHS="35-virtual;50-virtual;60-virtual;70-virtual;80-virtual"
+export CUDA_ARCH_LIST="sm_35,sm_50,sm_60,sm_70,sm_75,sm_80"
+export CUDAARCHS="35-virtual;50-virtual;60-virtual;70-virtual;75-virtual;80-virtual"
 
 # Only build the lowest non-deprecated arch to minimize build time
 if [[ "$target_platform" == "linux-ppc64le" || "$target_platform" == "linux-aarch64" ]]; then
-  export CUDA_ARCH_LIST="sm_60"
-  export CUDAARCHS="60-virtual"
+  export CUDA_ARCH_LIST="sm_60;sm_80"
+  export CUDAARCHS="60-virtual;80-virtual"
```

Comment:
```suggestion
  export CUDA_ARCH_LIST="sm_60"
  export CUDAARCHS="60-virtual"
```
Still only enough time for one arch on Travis.

Reply:
We could try [cross-compiling]( https://conda-forge.org/docs/maintainer/knowledge_base.html#cross-compilation ) if that is helpful

Reply:
Does CUDA 11 support cross-compile? If not, I need to wait for the second migrator for arm/ppc.

Reply:
Yep it does. Several packages are already building for CUDA 11 using cross-compilation. For example [`nccl` uses cross-compilation]( https://github.com/conda-forge/nccl-feedstock/blob/6cce7b8815dd20a86ab9bf15fa9b9526ba617503/conda-forge.yml#L6-L8 )

It also will be supported for CUDA 12 (once work on the other archs wraps up)